src/EventService.Data/EventUserRepository.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading.Tasks;
+using LT.DigitalOffice.EventService.Data.Interfaces;
+using LT.DigitalOffice.EventService.Data.Provider;
+using LT.DigitalOffice.EventService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.EventService.Data;
+
+  public class EventUserRepository : IEventUserRepository
+  {
+    private readonly IDataProvider _provider;
+
+    public EventUserRepository(IDataProvider provider)
+    {
+      _provider = provider;
+    }
+
+    public async Task<bool> IsUserAddedToEventAsync(Guid userId, Guid eventId)
```

Comment:
EventUserRepository (event+user); and you check existance of event+user =>
DoesExistAsync will be OK for methods' name